core/domain/user_jobs_one_off_test.py

```diff
@@ -2073,3 +2073,91 @@ def test_job_removes_deleted_exp_from_edited_explorations(self):
+    BROKEN_BASE64 = (
+        'data:imag64,/9j/4AAQSkZJRgABAQEASABIAAD//gATQ3JlYXRlZCB3aXRoIEdJTVD/2wBDAAEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQH/2wBDAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQH/wgARCAABAAEDAREAAhEBAxEB/8QAFAABAAAAAAAAAAAAAAAAAAAACf/EABQBAQAAAAAAAAAAAAAAAAAAAAD/2gAMAwEAAhADEAAAAX8P/8QAFBABAAAAAAAAAAAAAAAAAAAAAP/aAAgBAQABBQJ//8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAgBAwEBPwF//8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAgBAgEBPwF//8QAFBABAAAAAAAAAAAAAAAAAAAAAP/aAAgBAQAGPwJ//8QAFBABAAAAAAAAAAAAAAAAAAAAAP/aAAgBAQABPyF//9oADAMBAAIAAwAAABAf/8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAgBAwEBPxB//8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAgBAgEBPxB//8QAFBABAAAAAAAAAAAAAAAAAAAAAP/aAAgBAQABPxB//9k=')  # pylint: disable=line-too-long
+
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = user_jobs_one_off.VerifyProfilePictureOneOffJob.create_new()
+        user_jobs_one_off.VerifyProfilePictureOneOffJob.enqueue(job_id)
+        self.assertEqual(
+            self.count_jobs_in_mapreduce_taskqueue(
+                taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
+        self.process_and_flush_pending_mapreduce_tasks()
+        stringified_output = (
+            user_jobs_one_off.VerifyProfilePictureOneOffJob.get_output(job_id))
+        eval_output = [ast.literal_eval(stringified_item) for
+                       stringified_item in stringified_output]
+        return eval_output
+
+    def setUp(self):
+        def empty(*_):
+            """Function that takes any number of arguments and does nothing."""
+            pass
+
+        # We don't want to signup the superadmin user.
+        with self.swap(test_utils.TestBase, 'signup_superadmin_user', empty):
+            super(VerifyProfilePictureOneOffJobTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        user_services.generate_initial_profile_picture(self.owner_id)
+
+    def test_correct_profile_picture(self):
+        output = self._run_one_off_job()
+        self.assertEqual(output, [['CORRECT PROFILE PICTURE', 1]])
+
+    def test_profile_picture_non_standart_dimensions(self):
+        user_services.update_profile_picture_data_url(
+            self.owner_id, self.PNG_IMAGE_WRONG_DIMENSIONS_BASE64)
+        output = self._run_one_off_job()
+        self.assertEqual(
+            output, [['PROFILE PICTURE NON-STANDART DIMENSIONS - 150,160', 1]])
+
+    def test_cannot_load_profile_picture(self):
```

Comment:
If possible, test names should describe both the outcome and the conditions that led to it. In this case:

``test_invalid_image_has_cannot_load_picture_error``

Or similar is a bit clearer since it describes what state the test is using to lead to a particular outcome (in this case, the type of error) & what that outcome is.

Please update this & the names of the other new tests here.